modules/auxiliary/scanner/http/buildmaster_login.rb

```diff
@@ -0,0 +1,139 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::AuthBrute
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Inedo BuildMaster Login Scanner',
+      'Description' => %{
+          This module will attempt to authenticate to BuildMaster. There is a default user 'Admin'
+          which has the default password 'Admin'.
+      },
+      'Author'         => [ 'James Otten <jamesotten1[at]gmail.com>' ],
+      'License'        => MSF_LICENSE,
+      'DefaultOptions' => { 'VERBOSE' => true })
+    )
+
+    register_options(
+      [
+        Opt::RPORT(81),
+        OptString.new('USERNAME', [false, 'Username to authenticate as', 'Admin']),
+        OptString.new('PASSWORD', [false, 'Password to authenticate with', 'Admin'])
+      ], self.class
```

Comment:
`, self.class` is no longer required.

Reply:
I noticed that the same thing exists in the following files:
```
modules/auxiliary/scanner/http/binom3_login_config_pass_dump.rb
modules/auxiliary/scanner/http/epmp1000_web_login.rb
modules/auxiliary/scanner/http/meteocontrol_weblog_extractadmin.rb
modules/auxiliary/scanner/http/allegro_rompager_misfortune_cookie.rb
modules/auxiliary/scanner/http/epmp1000_cmd_exec.rb
modules/auxiliary/scanner/http/gavazzi_em_login_loot.rb
modules/auxiliary/scanner/http/epmp1000_dump_hashes.rb
modules/auxiliary/scanner/http/cisco_ssl_vpn_priv_esc.rb
modules/auxiliary/scanner/http/epmp1000_dump_config.rb
modules/auxiliary/scanner/http/elasticsearch_traversal.rb
```
Thoughts on a separate PR to change those?